gradle-plugin-testing-error-prone/src/main/java/com/palantir/gradle/testing/errorprone/GradlePluginTestHelpers.java

```diff
@@ -37,13 +42,56 @@ static boolean notWithinGradlePluginTests(Tree tree, VisitorState state) {
         return !WITHIN_GRADLE_PLUGIN_TESTS_CLASS.matches(tree, state);
     }
 
-    static boolean isGradlePluginTestsLibraryMethod(MethodInvocationTree tree, VisitorState state) {
+    static boolean notGradlePluginTestsLibraryMethod(MethodInvocationTree tree) {
         return Optional.ofNullable(ASTHelpers.getSymbol(tree))
                 .map(Symbol.MethodSymbol::enclClass)
                 .map(classSymbol -> classSymbol.getQualifiedName().toString())
                 .filter(className -> className.startsWith(LIBRARY_PACKAGE))
+                .isEmpty();
+    }
+
+    static Optional<ExpressionTree> findVariableInitializer(Symbol.VarSymbol var, VisitorState state) {
+        return Optional.ofNullable(ASTHelpers.findEnclosingNode(state.getPath(), MethodTree.class))
```

Comment:
Do you only want to find variables initialized in this method? Would it be desireable to find e.g. fields on the test class as well? 

https://github.com/google/error-prone/blob/690c2c1038bdbeb1b08cd278e68a176c9140e92d/core/src/main/java/com/google/errorprone/bugpatterns/nullness/NullnessUtils.java#L639-L650